.gitignore

```diff
@@ -59,14 +59,13 @@ deployments
 
 # lockfiles
 packages/core-types/package-lock.json
-packages/account-abstraction/package-lock.json
+packages/account/package-lock.json
 packages/common/package-lock.json
-packages/ethers-lib/package-lock.json
+packages/bundler/package-lock.json
 packages/node-client/package-lock.json
-packages/relayer/package-lock.json
-packages/smart-account/package-lock.json
-packages/transactions/package-lock.json
+packages/paymaster/package-lock.json
 package-lock.json
```

Comment:
can add particle and web3-auth package lock files to ignore

Reply:
done